sdk/examples/wallet/nft_collection/01_mint_collection_nft.rs

```diff
@@ -1,72 +1,96 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! In this example we will mint the 150 nfts with issuer feature.
-//! Rename `.env.example` to `.env` and run 01_create_wallet.rs before.
+//! In this example we will mint some collection NFTs with issuer feature.
 //!
-//! `cargo run --example mint_collection_nft --release`
+//! Make sure that `example.stronghold` and `example.walletdb` already exist by
+//! running the `create_wallet` example and  that you have created an Issuer NFT ID
+//! by running the `mint_issuer_nft` example!
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example mint_collection_nft
+//! ```
 
-use std::str::FromStr;
+use std::{env::var, str::FromStr};
 
 use iota_sdk::{
     types::block::{
         address::{Address, NftAddress},
         output::NftId,
+        payload::transaction::TransactionId,
     },
-    wallet::{MintNftParams, Result, Wallet},
+    wallet::{Account, MintNftParams, Result, Wallet},
 };
 
+// !!! Replace with the NFT address from the NFT we minted in `mint_issuer_nft` example !!!
+// const ISSUER_NFT_ID: &str = "0x13c490ac052e575cffd40e170c2d46c6029b8b68cdf0e899b34cde93d2a7b28a";
+const ISSUER_NFT_ID: &str = "0xe009d73b326b5e3c3430734695421d34fc0f820005181c987962d31daf25db82";
+// The NFT collection size
+const NFT_COLLECTION_SIZE: usize = 150;
+// Mint NFTs in chunks since the transaction size is limited
+const NUM_NFTS_MINTED_PER_TRANSACTION: usize = 50;
+// The location of the NFT.
+const NFT_URI: &str = "ipfs://wrongcVm9fx47YXNTkhpMEYSxCD3Bqh7PJYr7eo5Ywrong";
+// The name of the NFT
+const NFT_NAME: &str = "Shimmer OG NFT";
+// The description of the NFT
+const NFT_DESCRIPTION: &str = "The Shimmer OG NFT was handed out 1337 times by the IOTA Foundation to celebrate the official launch of the Shimmer Network.";
+// The issuer of the NFT
+const NFT_ISSUER_NAME: &str = "IOTA Foundation";
+// The name of the NFT collection
+const NFT_COLLECTION_NAME: &str = "Shimmer OG";
```

Comment:
Do you really think this is a better representation? It's a JSON payload after all and there can be also other fields we didn't set here https://github.com/iotaledger/tips/blob/main/tips/TIP-0027/tip-0027.md#nft-schema

Reply:
I think you're right. It should be a single JSON string constant, that can be modified in any desirable way.

Reply:
Done (I think ... please check again)